tests/js/test-utils.js

```diff
@@ -22,15 +22,14 @@ import { createTestRegistry } from './utils';
  * Renders the given UI into a container to make assertions.
  *
  * @see {@link https://testing-library.com/docs/react-testing-library/api#render}
- *
+ * @since 1.7.1
```

Comment:
Ah ha! `@see` is allowed it seems (unless this is a bug?) it only needs to be above `@since` ?

Reply:
Hmm, I guess so. Previously the use of `@see` tag was odd, but also I guess I just found them awkward; VSCode and the like will still auto-link, and needing an explicit tag for the URL seemed like a lot of extra work to me. But if you prefer it, we can make it work 😅